example/src/main/java/org/nervos/ckb/NervosDaoExample.java

```diff
@@ -277,6 +271,8 @@ private static Transaction generateClaimingFromDaoTx(
             Collections.singletonList(new byte[]{}),
             Collections.singletonList(new Witness(new byte[0], NERVOS_DAO_DATA, new byte[0])));
 
-    return tx.sign(Numeric.toBigInt(DaoTestPrivateKey));
+    // TODO: fix tx sign
```

Comment:
why leave a TODO here, can we complete it by new `Signer` class?

Reply:
I found that most examples construct scriptGroup with address, so I plan to refactor them after cleaning and abstracting address, otherwise I have to refactor them again then.